File: app/src/main/java/com/example/apiproject/java/network/APIService.java
package com.example.apiproject.java.network;

import com.example.apiproject.java.data.model.FlowerResponse;

import retrofit2.Call;
import retrofit2.http.GET;

public interface APIService {
    @GET("api/?key=17879196-53036d227ddd261b2254201a7&q=yellow+flowers&image_type=photo")
    Call<FlowerResponse> getFlower();
}

File: app/src/main/java/com/example/init/api/ApiService.kt
package com.example.init.api

import com.example.init.data.model.FlowerResponse
import retrofit2.Call
import retrofit2.http.GET

interface ApiService {
   @GET("api/?key=17879196-53036d227ddd261b2254201a7&q=yellow+flowers&image_type=photo")
   fun getFlower(): Call<FlowerResponse>
}